### PR TITLE
(#170766031) Feature of Reseting password

### PR DESCRIPTION
**Pull request by jajajaden01**
#### What does this PR do?
This pull request ensure that a user can reset his/her password and update the password

#### Description of Task to be completed?
 - create the components for sending email endpoint and update password using `back-end endpoint`
 - create a reducer state to handle both components
 - create tests for these components and mock data

#### How should this be manually tested?
You can pull the `develop` and run these following commands `npm install` and run `npm test`

#### What are the relevant pivotal tracker stories?
[#170766031](https://www.pivotaltracker.com/story/show/170766031)

#### Screenshots
- Find your account page
Screenshot from 2020-03-25 01-32-10

- Change your password page
Screenshot from 2020-03-25 01-35-58

- Coveralls and Test
Screenshot from 2020-03-24 04-31-06

### CODE REVIEW

File: src/components/ResetPassword/ResetPassword.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+//     height: 100vh;
+//     position: relative;
+    
+//     .container {
+//         width: 100%;
+//         display: flex;
+//         flex-direction: row;
+//         height: 100vh;
+//     }
+// }
```
Comment (luc-tuyishime) [inline src/components/ResetPassword/ResetPassword.scss]: remove comments

File: src/components/common/Form/Form.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+//     margin-right: -($xlarge-padding);
+//     margin-bottom: 30px !important;
+//     font-style: normal;
+//     font-weight: bold;
+//     font-size: 150%;
+//     text-align: center;
+//   }
+// }
+
+/* ==== Reset password Page =========================== */
```
Comment (luc-tuyishime) [inline src/components/common/Form/Form.scss]: Remove comments

File: src/components/common/Input/Input.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+//     position: absolute;
+//     margin-top: -10px;
+//     width: 210px;
+//     font-size: 14px;
+//     color: $white;
+//     padding: 10px 5px;
+//     z-index: 9;
+//     background-color: $background-color;
+//   }
+// }
```
Comment (luc-tuyishime) [inline src/components/common/Input/Input.scss]: Please clean code.

Comment (jajajaden01) [reply src/components/common/Input/Input.scss]: yes, let me clean it

**Pull request merged**